docs/DeveloperGuide.md

```diff
@@ -806,41 +812,162 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the `CampusConnect.jar` file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Double-click the `CampusConnect.jar` file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+2. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### Adding a person
+
+1. Adding a `Person`:
+
+    1. **Test case:** `add n/John Doe p/98765432 e/johndoe@gmail.com a/John street, block 123, #01-01`<br>
+       **Expected:** A new `Person` is successfully created with name "John Doe", phone "98765432", email "johndoe@gmail.com" and address "John street, block 123, #01-01". Success details shown in the status message. Moreover, this new `Person` is visible in CampusConnect.
+
+    2. **Test case:** `add n/John Doe p/98765432 e/johndoe@gmail.com a/`<br>
+       **Expected:** No new `Person` is created. Error details shown in the status message.
```

Comment:
About your **Expected** description, while the first sentence is complete, the second one is not full. Is that what you intentionally do?

If not, you can consider change it and the rest

Reply:
i do personally believe that there is no need to elaborate further on what `Person` should have been created here since the expected output is that it will never be created. thus, i omitted the nitty gritty details of the supposed `Person` object that should be created to be succinct